Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,131 @@
+
+                //calculate and print bmi
+                float bmi = CalculateBodyMassIndex(weight, height);
+                Console.WriteLine(firstName + " " + lastName + " has a BMI of " + bmi + ".");
+            }
+        }
+
+        public static string PromptString(string checkString)
+        {
+            Console.Write(checkString);
+            string returnString = Console.ReadLine();
+            if (returnString != "")
+            {
+                return returnString;
+            }
+            else
+            {
+                Console.WriteLine("Name cannot be empty.");
+                return "-";
+            }
+        }
+
+        public static int PromptInt(string checkString)
+        {
+            Console.Write(checkString);
+            string returnString = Console.ReadLine();
+            if (returnString == "")
+            {
+                return 0;
+            }
+            else
+            {
+                bool isNumber = int.TryParse(returnString, out int returnInt);
+                if (!isNumber)
+                {
+                    Console.WriteLine(returnString + " is not a valid number.");
+                    return -1;
+                }
+                else
+                {
+                    return returnInt;
+                }
+            }
+        }
+        public static float PromptFloat(string checkString)
+        {
+            Console.Write(checkString);
+            string returnString = Console.ReadLine();
+            if (returnString == "")
+            {
+                return 0;
+            }
+            else
+            {
+                bool isNumber = float.TryParse(returnString, NumberStyles.Float, CultureInfo.InvariantCulture, out float returnFloat);
+                if (!isNumber)
+                {
+                    Console.WriteLine(returnString + " is not a valid number.");
+                    return -1;
+                }
+                else
+                {
+                    return returnFloat;
+                }
+            }
+        }
+
+        public static float CalculateBodyMassIndex(float weight, float height)
+        {
+            bool isWeightValid = true;
+            bool isHeightValid = true;
+
+            if (weight == 0 || weight == -1)
+            {
+                isWeightValid = false;
+            }
+
+            if (height == 0 || height == -1)
+            {
+                isHeightValid = false;
+            }
+
+            if (!isWeightValid || !isHeightValid)
+            {
+                return InvalidBMI(isWeightValid, isHeightValid, weight, height);
+            }
+
+            else
+            {
+                return weight / (height * height);
+            }
+        }
+
+        public static float InvalidBMI(bool isWeightValid, bool isHeightValid, float weight, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'InvalidBMI' to match pascal case naming rules, consider using 'InvalidBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4917199)